### DISCUSSION TITLE
Welcome to semanticClimate Discussions!

### DISCUSSION THREAD

**Discussion by petermr** [petermr/semanticClimate#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by petermr** [comment]
# Keeping records

Every chapter owner must record in sufficient detail for **others to repeat or verify the operation**. **IF THERE IS NO REPORT , THEN THE OPERATION HAS NOT BEEN DONE.** Record(in `semanticClimate` repository discussions):

* the operations on the raw data (e.g.converting PDF toHTML)
* creation of per chapter dictionaries
* editing of dictionaries
* annotation of HTML

**Comment by mrchristian** [comment]
Please note the discussion board has now moved to: https://github.com/orgs/semanticClimate/discussions
